### PR TITLE
Add `is.inRange()`

### PR DESCRIPTION
**Pull request by glhrmv**

### CODE REVIEW

**Comment by sudo-suhas** [comment]
Destructuring isn't supported on node 4.

**Comment by sindresorhus** [comment]
Yes

**Comment by sindresorhus** [comment]
Let’s call it `inRange`. And let’s do the Math.min/max thing you suggested so the range parameters can be specified in any order and also be negative.

**Comment by glhrmv** [comment]
How's this?

**Comment by brandon93s** [comment]
Could also borrow from [sindresorhus/in-range](https://github.com/sindresorhus/in-range/blob/master/index.js). It allows start to be optional, with a default of 0.

File: test.js [hunk 1]
```diff
@@ -293,3 +293,10 @@ test('is.typedArray', t => {
 	t.false(m.typedArray([]));
 	t.false(m.typedArray({}));
 });
+
+test('is.inRange', t => {
+	const x = 3;
+	const arr = [0, 5];
```
Comment (sindresorhus) [inline test.js]: You can just put this inline in the assertion.

I'd also like to see assertions for `[5, 0]`, `[-5, 5]`, `[5, -5]`.

**Comment by sindresorhus** [comment]
> Could also borrow from sindresorhus/in-range. It allows start to be optional, with a default of 0.

Good idea. I forgot I had that module. We could also support `is.inRange(5, 10)`. So the user can either use an array as range or a number as upper-bound. Thoughts?

**Comment by brandon93s** [comment]
I agree. The flexibility of supporting both inputs would be nice.

File: test.js (hunk 1 above)
Comment (gioragutt) [reply test.js]: Also add assertions for `is.inRange` that return `false`.

I also agree with inlining the test data.

**Comment by glhrmv** [comment]
How about now? 😃

**Comment by sindresorhus** [comment]
Perfect! Thanks for working on this @glhrmv 🙌

**Comment by glhrmv** [comment]
@sindresorhus Thank you! It was my pleasure.

**Pull request merged**